skytap/resource_skytap_network.go

```diff
@@ -83,7 +83,7 @@ func resourceSkytapNetworkCreate(d *schema.ResourceData, meta interface{}) error
 	}
 
 	log.Printf("[INFO] network create")
```

Comment:
include id

Reply:
There is no ID at this stage. The ID is returned after the create and is printed out at that point - see line 98.